lib/seed/stub/mjs.stub

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('table_name').del()
```

Comment:
why not use async/await?

Reply:
This file is in cjs format, while it should be in esm... good catch.